app/lib/frontend/handlers/pubapi.dart

```diff
@@ -252,3 +253,22 @@ class PubApi {
   Future<Response> packageJson(Request request, String package) =>
       packageShowHandlerJson(request, package);
 }
+
```

Comment:
What does this is do and why, this critically needs to be explained in a doc-comment :)